exchange/bidder.go

```diff
@@ -178,6 +225,26 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 	return seatBid, errs
 }
 
+func getNativeImpByImpID(impID string, request *openrtb.BidRequest) (*openrtb.Native, error) {
+	for _, impInRequest := range request.Imp {
+		if impInRequest.ID == impID && impInRequest.Native != nil {
+			return impInRequest.Native, nil
+		}
+	}
+	return nil, errors.New("Could not find native imp")
+}
+
+func getAssetByID(id int64, assets []nativeRequests.Asset) nativeRequests.Asset {
+	var asset nativeRequests.Asset
+	for i := 0; i < len(assets); i++ {
```

Comment:
Consider using the `range` syntax of `for`.

Reply:
Done